src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -496,15 +501,11 @@ public EmbeddedPostgres start() throws IOException
 
     private static List<String> system(String... command)
     {
-        return system(ProcessBuilder.Redirect.INHERIT, ProcessBuilder.Redirect.INHERIT, command);
-    }
-
-    private static List<String> system(ProcessBuilder.Redirect errorRedirector, ProcessBuilder.Redirect outputRedirector, String... command)
-    {
         try {
             final ProcessBuilder builder = new ProcessBuilder(command);
-            builder.redirectError(errorRedirector);
-            builder.redirectOutput(outputRedirector);
+            builder.redirectError(ProcessBuilder.Redirect.PIPE);
+            builder.redirectOutput(ProcessBuilder.Redirect.PIPE);
```

Comment:
If output is voluminous, this runs the risk of taking up a lot of memory (might any internal buffers get overflow'd?).  But we just use this for small control commands.  So we're probably ok?

Reply:
The pipe buffer should be fixed size `O(KB)` and if too much data gets written it will block.  Our reader is a separate thread and should be nearly always available to take data.  I don't think there's much risk here.